### DISCUSSION TITLE
希望加入检测功能，云盘里有的文件还要继续上传一遍

### DISCUSSION THREAD

**Discussion by zzerosama** [messense/aliyundrive-webdav#500] [Ideas]
希望加入检测功能，云盘里有的文件还要继续上传一遍，用的群晖Cloud Sync，同步视频，有一点改动，就要全再上传一遍，一直停不下来，记得有一个叫zx5253的大佬做的docker版就不会，可惜现在他已经不更新了，网盘里存在相同文件时，他的就会自动合并跳过

**Comment by messense** [comment]
个人没有上传需求不会投入时间，需要的话请考虑[技术支持](https://github.com/messense/aliyundrive-webdav/discussions/442)
